atd-vzd/schema/atd_txdot_crashes.sql

```diff
@@ -219,84 +216,209 @@ CREATE TABLE public.atd_txdot_crashes (
+
+
+--
+-- Name: atd_txdot_crashes_geocode_provider_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
 CREATE INDEX atd_txdot_crashes_geocode_provider_index ON public.atd_txdot_crashes USING btree (geocode_provider);
 
 
 --
--- TOC entry 5629 (class 1259 OID 2346768)
--- Name: atd_txdot_crashes_geocode_status_index; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes_geocode_status_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
 CREATE INDEX atd_txdot_crashes_geocode_status_index ON public.atd_txdot_crashes USING btree (geocode_status);
 
 
 --
--- TOC entry 5630 (class 1259 OID 2346769)
--- Name: atd_txdot_crashes_geocoded_index; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes_geocoded_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
 CREATE INDEX atd_txdot_crashes_geocoded_index ON public.atd_txdot_crashes USING btree (geocoded);
 
 
 --
--- TOC entry 5631 (class 1259 OID 2346770)
--- Name: atd_txdot_crashes_is_retired_index; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes_investigat_agency_id_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_crashes_investigat_agency_id_index ON public.atd_txdot_crashes USING btree (investigat_agency_id);
+
+
+--
+-- Name: atd_txdot_crashes_is_retired_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
 CREATE INDEX atd_txdot_crashes_is_retired_index ON public.atd_txdot_crashes USING btree (is_retired);
 
 
 --
--- TOC entry 5632 (class 1259 OID 2346771)
--- Name: atd_txdot_crashes_qa_status_index; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes_original_city_id_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_crashes_original_city_id_index ON public.atd_txdot_crashes USING btree (original_city_id);
+
+
+--
+-- Name: atd_txdot_crashes_position_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_crashes_position_index ON public.atd_txdot_crashes USING gist ("position");
+
+
+--
+-- Name: atd_txdot_crashes_qa_status_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
 CREATE INDEX atd_txdot_crashes_qa_status_index ON public.atd_txdot_crashes USING btree (qa_status);
 
 
 --
--- TOC entry 5633 (class 1259 OID 2346795)
--- Name: idx_atd_txdot_crashes_crash_id; Type: INDEX; Schema: public; Owner: atd_vz_data
+-- Name: atd_txdot_crashes_sus_serious_injry_cnt_index; Type: INDEX; Schema: public; Owner: visionzero
+--
+
+CREATE INDEX atd_txdot_crashes_sus_serious_injry_cnt_index ON public.atd_txdot_crashes USING btree (sus_serious_injry_cnt);
+
+
+--
+-- Name: atd_txdot_crashes_temp_record_index; Type: INDEX; Schema: public; Owner: visionzero
 --
 
-CREATE UNIQUE INDEX idx_atd_txdot_crashes_crash_id ON public.atd_txdot_crashes USING btree (crash_id);
+CREATE INDEX atd_txdot_crashes_temp_record_index ON public.atd_txdot_crashes USING btree (temp_record);
```

Comment:
This freaked me out for a second because of the unique index is removed here. But, it is fulfilled by the setting of `crash_id` as the primary key, which will get its own `btree` index.